### PR TITLE
Update Report dto

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 1]
```diff
@@ -1639,7 +1639,7 @@ public Report getSubReport(String subReportId, Set<String> severityLevels) {
 
     @Transactional(readOnly = true)
     public List<Report> getNodeReport(UUID nodeUuid, String reportId, ReportType reportType, Set<String> severityLevels) {
-        return reportService.getReport(UUID.fromString(reportId), nodeUuid.toString(), nodeUuid + "@" + reportType.reportKey, ReportNameMatchingType.EXACT_MATCHING, severityLevels);
+        return reportService.getReport(UUID.fromString(reportId), nodeUuid.toString(), null, null, severityLevels);
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:1642]: Why do you change this behavior ?

Comment (ayolab) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java:1642]: no need for filtering here we already call this endpoint with the right report ID we want to access (the one related to NETWORK_MODIFICATIONS / COMPUTATIONS)

**Review by TheMaskedTurtle — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 2]
```diff
@@ -1639,7 +1639,7 @@ public Report getSubReport(String subReportId, Set<String> severityLevels) {
 
     @Transactional(readOnly = true)
     public List<Report> getNodeReport(UUID nodeUuid, String reportId, ReportType reportType, Set<String> severityLevels) {
-        return reportService.getReport(UUID.fromString(reportId), nodeUuid.toString(), nodeUuid + "@" + reportType.reportKey, ReportNameMatchingType.EXACT_MATCHING, severityLevels);
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:1642]: reportType is not used anymore: can be remove from here, rest api, and front-end

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: some clean can be done
tests: on going

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Pull request merged**